src/swarm/neo/client/mixins/Controllers.d

```diff
@@ -303,25 +311,23 @@ template Controllers ( )
 
         /***********************************************************************
 
-            This method of `ISuspendable` cannot currently be implemented and
-            thus always throws. To implement this method correctly, this class
-            would need to somehow hook into the notifications of the request
-            being controlled, in order to receive the suspended/resumed
-            notifications to know when the request had actually changed state
-            (on the nodes' side).
+            TODO add description
```

Comment:
What did you have in mind here? I'm not sure if we need to say anything beyond what Returns states now.